### ISSUE TITLE
KMS in React Native

### ISSUE THREAD

**Issue by Cancuuu** [tatumio/tatum-kms#36]
Thats a simple question, can i use tatum-kms in react native/expo ? there are another packages that i have to install?

**Comment by kahwooi** [comment]
@Cancuuu there is no reason to run Tatum KMS in mobile app, because the main purpose of Tatum KMS is to securely manage the private keys, so that the private keys/mnemonic are not exposed to the outside world, which includes the mobile app.

The Tatum KMS should be run on a server with a Deny-From-All environment to meet the highest security standards, and when in daemon mode it should just listen to pending transactions from Tatum Cloud and sign those transactions with the private keys stored in the Tatum KMS.

In summary the purposes of Tatum KMS are:
1. Create blockchain private keys and addresses and store them in KMS as CLI tools.
2. Listen to pending transactions from Tatum Cloud and sign those transactions in Daemon mode.
